docs/hive-integration/platform-selection.md

```diff
@@ -0,0 +1,16 @@
+# Platform Selection
+
+The Assisted Installer currently supports the following OpenShift platforms:
+- BareMetal
+- VSphere
+- None
+
+Select the platform in the AgentClusterInstall CR, via `spec.platformType`.
+
+In some cases, the system will automatically set the platform type:
+- The default platform for multi-node clusters is be BareMetal
+- The default platform for Single Node OpenShift is None
+- Enabling `spec.networking.userManagedNetworking` will cause the platform to be None
```

Comment:
This is not true. In vsphere, it still be vsphere platform, just without the networking

Reply:
At least I hope that this is what we implemented

Reply:
Yes, I meant that this is the behavior in case the platform isn't specified.  Fixed.